src/Tribe/Models/Post_Types/Event.php

```diff
@@ -46,15 +52,13 @@ protected function build_properties( $filter ) {
 
 			$post_id = $this->post->ID;
 
-			$post_meta = get_post_meta( $post_id );
-
-			$start_date            = isset( $post_meta['_EventStartDate'][0] ) ? $post_meta['_EventStartDate'][0] : null;
-			$start_date_utc        = isset( $post_meta['_EventStartDateUTC'][0] ) ? $post_meta['_EventStartDateUTC'][0] : null;
-			$end_date              = isset( $post_meta['_EventEndDate'][0] ) ? $post_meta['_EventEndDate'][0] : null;
-			$end_date_utc          = isset( $post_meta['_EventEndDateUTC'][0] ) ? $post_meta['_EventEndDateUTC'][0] : null;
-			$duration              = (int) isset( $post_meta['_EventDuration'][0] ) ? $post_meta['_EventDuration'][0] : null;
+			$start_date            = get_post_meta( $post_id, '_EventStartDate', true );
+			$start_date_utc        = get_post_meta( $post_id, '_EventStartDateUTC', true );
+			$end_date              = get_post_meta( $post_id, '_EventEndDate', true );
+			$end_date_utc          = get_post_meta( $post_id, '_EventEndDateUTC', true );
+			$duration              = get_post_meta( $post_id, '_EventDuration', true );
 			$timezone_string       = Timezones::get_event_timezone_string( $post_id );
-			$all_day               = tribe_is_truthy( isset( $post_meta['_EventAllDay'][0] ) ? $post_meta['_EventAllDay'][0] : null );
+			$all_day               = tribe_is_truthy( get_post_meta( $post_id, '_EventAllDay', true ) );
```

Comment:
Why did we change this?

Reply:
https://github.com/the-events-calendar/the-events-calendar/pull/3833/files/470c7f19ca6ddc8caf94940ca7b807c306224f66#diff-8d783ab4e9c9e65f92a01f9ba840456d5b38559ca6c3650f42969c36d7be60e3

Reply:
@lucatume I would like us to revert to the old way here, I don't think it's worth the refactor here. Since the reason why we changed in the past was due to some problems with performance.

Reply:
After some Slack conversation, we agreed on going back to the `get_post_meta( $post_id )` version. A bit less readable, but the change was made for a performance gain.

Reply:
Prior PR for ref: https://github.com/the-events-calendar/the-events-calendar/pull/2987